documentContentProviders/dockerInspect.ts

```diff
@@ -0,0 +1,22 @@
+import { TextDocumentContentProvider, Uri, window, workspace } from "vscode";
+import { docker } from "../commands/utils/docker-endpoint";
+
+export const IMAGE_DOMAIN = "image";
+export const SCHEME = "docker-inspect";
+export const URI_EXTENSION = ".json";
```

Comment:
I guess its ok, the only thing you lose is the ability to make a change and then reload the window in the running instance to pick up that change, without having to restart the debugger. I typically restart the debugger anyway so it isn't a big deal.

Otherwise, changes look good!